packages/mobile/android/app/build.gradle

```diff
@@ -129,6 +129,7 @@ android {
         testBuildType System.getProperty('testBuildType', 'debug')
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         resValue "string", "build_config_package", "org.celo.mobile"
+        missingDimensionStrategy 'react-native-camera', 'mlkit'
```

Comment:
Can we get a rough estimate of how much this mlkit affects the apk size?
Here are the commands to build an apk, from the `android` folder:
```
./gradlew clean
./gradlew bundleReleaseJsAndAssets 
./gradlew assembleRelease -x bundleReleaseJsAndAssets
```